components/CommentModal/CommentModal.js

```diff
@@ -5,14 +5,21 @@ import { useProfile, useMember } from "../db"
 import { useEditTestimony } from "../db/testimony/useEditTestimony"
 import EmailToMyLegislators from "./EmailToMyLegislators"
 import EmailToCommittee from "./EmailToCommittee"
+import Tweet from "./Tweet"
+
+const testimonyEmailAddress = "archive@digitaltestimony.com"
+const webSiteBillAddress = "https://digital-testimony-dev.web.app/bill?id="
```

Comment:
We should use `location.origin` instead of `https://digital-testimony-dev.web.app`, so it always points to the same site the user is on.

Reply:
Good idea.  I used window.location.href as mentioned above.